openshift/e2e-common.sh

```diff
@@ -135,7 +135,8 @@ function install_catalogsource(){
 
   source ./test/lib.bash
   create_namespaces || exit $?
-  ensure_catalogsource_installed || exit $?
+  # Do not set OPENSHIFT_CI to use nightly build images.
+  OPENSHIFT_CI="" ensure_catalogsource_installed || exit $?
```

Comment:
I don't think the nightlies will always work with the pinned version above, will they?

Reply:
You are correct.

So I updated to use serverless-operator's master branch.
The pinned version was removed but I changed the script to use `yq` command to modify CSV instead of patch command, so the configmap patch problem does not happen.